logict.cabal

```diff
@@ -51,5 +51,6 @@ test-suite logict-tests
     logict -any,
     mtl >=2 && <2.3,
     tasty,
-    tasty-hunit
+    tasty-hunit,
+    tasty-expected-failure >= 0.12 && < 0.13
```

Comment:
Any reason to put such tight bounds? What is missing from `tasty-expected-failure-0.11`?

Reply:
Version 0.12 is the minimum version to properly handle timeouts.  Previously to that, the timeout was not caught as an expected failure and caused actual failure in the test run.

Reply:
I see. I'd prefer to remove the upper bound nevertheless.

Reply:
Removed.